static/js/components/dashboard/util.js

```diff
@@ -16,3 +17,26 @@ export const courseListToolTip = (text: string, id: string) => (
     </ReactTooltip>
   </div>
 );
+
+export const isCurrentlyEnrollable = (enrollmentStartDate, now) => (
+  enrollmentStartDate !== null &&
+    enrollmentStartDate !== undefined &&
+    enrollmentStartDate.isSameOrBefore(now || moment(), 'day')
+);
+
+export const formatGrade = (grade) => {
+  if (_.isNil(grade) || grade === '') {
+    return '';
+  } else {
+    let gradeString;
+    if (_.isString(grade)) {
+      gradeString = _.round(Number(grade));
+    } else if (_.isInteger(grade)) {
+      gradeString = grade;
```

Comment:
fair enough. i was aiming to avoid unnecessary operations, but the performance is quite unlikely to take a hit